astropy/units/tests/test_quantity_non_ufuncs.py

```diff
@@ -1766,9 +1766,9 @@ def check2(self, function, *args, **kwargs):
     @pytest.mark.parametrize(
         "kwargs",
         (
-            dict(return_index=True, return_inverse=True),
```

Comment:
Just having a very quick look, but I am not in favour of changes like these: this is testing different inputs to functions, and it really helps readability to have the test look like those inputs, i.e., `dict(key=value)` is very much used on purpose here.

I guess I am with @pllim that we should focus here on just replacement for speed reasons, not more stylistic changes.

(Note that I do confirm that python is slightly slower, but I would suggest one's effort should then be to improve python such that it is the same speed rather than change codebases; also it is measured in 10s of ns, so really is not going to matter in all but critical paths.)

Reply:
Done.